### ISSUE TITLE
Document where users can submit feedback on the VEDA JupyterHub (GitHub discussions?)

### ISSUE THREAD

**Issue by maxrjones** [NASA-IMPACT/veda-jupyterhub#55]
Is there already a standard place for VEDA JupyterHub users to submit feedback on their experiences? If not, could we enable discussions on this repo to be that place? I'd like to include a note in https://github.com/NASA-IMPACT/veda-docs/pull/165 (or a follow-up PR) about how to give feedback on the VS Code integration.

**Comment by batpad** [comment]
@maxrjones I think that's a great idea!

Just want to check if folks feel things are getting fragmented or so - @j08lue @freitagb would you folks have thoughts or opinions on enabling Discussions in this repository and using that to gather user feedback and discuss questions around the VEDA Jupyterhub that maybe a bit different from "Issues".

@maxrjones if there's no objections, let's connect and I can go ahead and enable Discussions here.

**Comment by j08lue** [comment]
Yes! Ways of getting support around VEDA services could use better documentation. 🙏 

@wildintellect has earlier led a spike on choosing and setting up a central support system, possibly across MAAP and VEDA (and maybe more).

- https://github.com/NASA-IMPACT/veda-architecture/issues/255

I think it was not conclusive, though, and we continued to use MAAP Slack for first-level support and tickets on various repos for second-level, @wildintellect?

Perhaps a good occasion to revisit this? GitHub Discussions on a central repo or several ones was one of the options considered.

**Comment by j08lue** [comment]
We had big questions - [this](https://developmentseed.slack.com/archives/C03LKRDS22W/p1683731140573909) was one of mine...

> The main advantage of Discourse, in my view, would be that it can be even more public, since users can log on with various accounts (Google, GitHub, local accounts, …).
> To me, the decisive question is: Do we want to try to establish a completely public and prominent place to ask Earth science compute support questions at NASA, across MAAP, EIS, VEDA, possibly even advertised for [NASA Openscapes](https://nasa-openscapes.github.io/) audiences etc?
> Then GitHub Discussions feels too small.
> However, if we just want to bring together EIS, MAAP, maybe VEDA and GHG Center in the future, GitHub Discussions - a single or several instances - is probably fine.

I we tabled this in the past, because we did not have a lot of users we needed to support, besides EIS and MAAP (both using MAAP).

**Comment by maxrjones** [comment]
Thanks for this context! If I'm interpreting those threads correctly, people were leaning towards a Discourse server that spans multiple NASA-funded groups (e.g., VEDA, MAAP, CryoCloud, OpenScapes, possibly even TOPS) which would take coordination with other group leaders. So, let's table this issue as pending an outcome from https://github.com/NASA-IMPACT/veda-architecture/issues/255. I will comment there about my interest in moving that forward.

**Comment by batpad** [comment]
@maxrjones 👍 and if you need something more immediately to refer users to, I think can always ask them to make issues in this repository with any problems they are facing or questions.
